api/hasura/event-triggers/utils/handleCheckNomineeMsg.ts

```diff
@@ -0,0 +1,30 @@
+import { gql } from '../../../../api-lib/Gql';
+import { sendSocialMessage } from '../../../../api-lib/sendSocialMessage';
+import { EventTriggerPayload } from '../../types';
+
+export default async function handleCheckNomineeMsg(
```

Comment:
this file should be outside of the event-triggers folder since its not actually an event trigger itself.